spec/syntax.md

```diff
@@ -112,7 +134,7 @@ hello.format()
 
 ### Simple Placeholders
 
-Messages may contain placeholders within inner `{…}` delimiters,
+Messages can contain ***placeholders*** within inner `{…}` delimiters,
```

Comment:
I'm not sure if you need to emphasize `placeholders` here, since we're still in the _Overview_ section. Up to you.

Reply:
Probably correct. I think the ***term*** should actually appear only in a definition, e.g. 

> A ***placeholder*** is a...

Which the *placeholder* mentions eventually link to (replacing the ***highlighting*** with `<dfn id="term">` in the output HTML)